x-pack/plugins/secops/public/components/timeline/body/renderers/empty_column_renderer.test.tsx

```diff
@@ -56,14 +54,22 @@ describe('empty_column_renderer', () => {
   });
 
   test('should return an empty value', () => {
-    delete mockDatum.source;
+    mockDatum = deleteItem(mockDatum, findItemIdx(mockDatum, 'source.ip'));
     const emptyColumn = emptyColumnRenderer.renderColumn({
       columnName: 'source.ip',
-      eventId: mockDatum._id,
-      value: null,
+      eventId: _id,
+      values: null,
       field: defaultHeaders.find(h => h.id === 'source.ip')!,
     });
     const wrapper = mount(<span>{emptyColumn}</span>);
     expect(wrapper.text()).toEqual(getEmptyValue());
   });
 });
+
+const findItemIdx = (data: TimelineNonEcsData[], field: string): number =>
+  data.findIndex(d => d.field === field);
+
+const deleteItem = (data: TimelineNonEcsData[], idx: number) => [
+  ...data.slice(0, idx),
+  ...data.slice(idx + 1),
+];
```

Comment:
Are these names flipped? `deleteItem` has the idx but `findItemIdx` as the field as a string?

Did you want this?
```ts
findItem
deleteItemIdx
```